src/errors/Messages.js

```diff
@@ -112,6 +112,9 @@ const Messages = {
 
   MEMBER_FETCH_NONCE_LENGTH: 'Nonce length must not exceed 32 characters.',
 
+  GLOBAL_COMMAND:
```

Comment:
This key name doesn't really represent its content since it's about global command's permissions rather than global commands themselves:
```suggestion
  GLOBAL_COMMAND_PERMISSIONS:
```